from_cpython/Lib/test/test_complex.py

```diff
@@ -476,8 +474,8 @@ def test_repr(self):
         self.assertEqual(repr(complex(0, -INF)), "-infj")
         self.assertEqual(repr(complex(0, NAN)), "nanj")
 
-    def test_neg(self):
-        self.assertEqual(-(1+6j), -1-6j)
+    # def test_neg(self):
+    #     self.assertEqual(-(1+6j), -1-6j)
 
```

Comment:
This is `libpypa` bug, Already solved in local repository, and create a PR https://github.com/vinzenz/libpypa/pull/47. Maybe libpypa will use another solution, but it will be solved at last. When it done, re-enable it.